tables-test-fixtures/src/main/java/com/linkedin/openhouse/tablestest/HouseTablesH2Repository.java

```diff
@@ -11,4 +14,13 @@
  */
 @Repository
 @Primary
-public interface HouseTablesH2Repository extends HouseTableRepository {}
+public interface HouseTablesH2Repository extends HouseTableRepository {
+  Optional<HouseTable> findByDatabaseIdIgnoreCaseAndTableIdIgnoreCase(
+      String databaseId, String tableId);
+
+  @Override
+  default Optional<HouseTable> findById(HouseTablePrimaryKey houseTablePrimaryKey) {
```

Comment:
these changes are needed to ensure local test also has case insensitivity enabled.